packages/docs/docs/development/css+styleguide.md

```diff
@@ -0,0 +1,61 @@
+# CSS styleguide for Team Arbeidsflate
+
+This minimalistic style guide is designed to ensure consistency and scalability in our frontend projects utilizing CSS Modules. By adhering to these guidelines, we aim to improve maintainability and readability across our codebase.
```

Comment:
A bit wordy. Can we cut it down to a single sentence?

Reply:
Men det var jo favorittsetningen min!!

Reply:
Eh, ok 😅

Var mest generelt sett at jeg ville ha det så konsist som mulig slik at det er raskt å skumme.